app/grandchallenge/blogs/filters.py

```diff
@@ -0,0 +1,16 @@
+from django_filters import FilterSet, ModelMultipleChoiceFilter
+from django_select2.forms import Select2MultipleWidget
+
+from grandchallenge.blogs.models import Post, Tag
+from grandchallenge.core.filters import FilterForm
+
+
+class BlogFilter(FilterSet):

```

Comment:
I like this implementation! It also means that we could add author filtering (but be really careful about that! filter the queryset by the users that are authors first), or simple text search of the content.

Reply:
Yeah, filtering by authors is not a bad idea. Should I go ahead and add this or would you rather leave it at the tag filter for now?